stdlib/2/formatter.pyi

```diff
@@ -0,0 +1,104 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/formatter.py
+from typing import Any, IO, List, Optional, Tuple
+
+AS_IS = None
+FONT_TYPE = Tuple[str, bool, bool, bool]
+STYLES_TYPE = Tuple[Any, ...]
+
+class NullFormatter:
+    writer = ...  # type: Any
+    def __init__(self, writer: Any = None) -> None: ...
```

Comment:
writer should be of type `Optional[NullWriter]` I believe. The default should be `...`.